### ISSUE TITLE
Accessibility-> Missing form label on Enter data landing page

### ISSUE THREAD

**Issue by kgorana** [bcgov/bcparks-ar-api#258]
## Description
**How to replicate:**
* Navigate to the A&R test environment and log in: [A&R Admin](https://d3ahrzu0mpndj3.cloudfront.net/enter-data)
* Check if form labels are present on page components.

**Issue:**
* Missing form labels on the “Date” field. Please make sure that the fields also have form labels when fixing this issue.

  ![Image](https://github.com/bcgov/bcparks-ar-api/assets/145403218/5e912c08-67e8-48a7-9a3b-0c8239237ccf)

## Linked issues
relates to
- [x] ~[BRS-822](https://bcparksdigital.atlassian.net/browse/BRS-822)~
